packages/types/src/ethermint/crypto/v1/ethsecp256k1/keys.ts

```diff
@@ -1,155 +0,0 @@
-/* eslint-disable */
```

Comment:
The `ethsecp256k1` structure is identical to `secp256k1` and is primarily utilized within `ethermint` for address generation and signing algorithms. Since these functions are now handled by the wallet, it is safe to remove this file from desmjs.